package-lock.json

```diff
@@ -3979,6 +3979,14 @@
       "integrity": "sha1-w2GT3Tzhwu7SrbfIAtu8d6gbHA8=",
       "dev": true
     },
+    "monotone-convex-hull-2d": {
```

Comment:
> Anyway, this is auto-generated, not sure if it is worthy to review it TBH.

It is. If only to detect when it's missing (like in #400). But also in case it differs from what the `package.json` says (like here; or if you use `git add -p` or equivalent to selectively include dependencies in your `package.json` when crafting your commits, etc.)